### DISCUSSION TITLE
knex error or 504 response [May 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2263] [Discord Threads]
> 🙋‍♂️ **Steffano**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can u show the full error message?  
#  
> 🙋‍♂️ **wkw**  
#### and when do you see that error? when creating a new project?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979562574557364294)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
